devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -23,100 +23,93 @@ metadata {
 		capability "Light"
 
 		// Generic
-		fingerprint profileId: "0104", deviceId: "0101", inClusters: "0006, 0008", deviceJoinName: "Light" //Generic Dimmable Light
+		fingerprint profileId: "0104", deviceId: "0101", inClusters: "0006, 0008", deviceJoinName: "Generic Dimmable Light"
```

Comment:
This pull request cannot contain any changes in other fingerprints. Please add Your fingerprint on an updated branch.